examples/chat.rs

```diff
@@ -118,6 +118,13 @@ fn main() {
         libp2p::Swarm::new(transport, behaviour, local_peer_id)
     };
 
+    if let Some(to_ban) = std::env::args().nth(1) {
```

Comment:
I suppose you put this to test if it worked, but I don't think it should be part of this example.